sim_learn.py

```diff
@@ -56,6 +53,8 @@ def main(args):
         stderr.write('                  The number total learning attempts is restarts*instances.\n')
         stderr.write('   seed_instance: random seed used for LTF array instance\n')
         stderr.write('      seed_model: random seed used for the model in first learning attempt\n')
+        stderr.write('        log_name: path to the logfile which contains results from all instances. The tool '
+                                        'will add a ".log" to log_name.\n')
```

Comment:
label this as optional

Reply:
https://github.com/nils-wisiol/pypuf/blob/e1d408389cff95a58dbaeb7cc39e702b99b6d4be/sim_learn.py#L56-L57